src/input.rs

```diff
@@ -47,6 +48,7 @@ impl Input {
             keys_hit: Vec::new(),
             mouse_moves: Vec::new(),
             mouse_moves_ndc: Vec::new(),
+            mouse_moves_raw: Vec::new(),
```

Comment:
What's the difference between `mouse_moves` and `mouse_moves_raw`?

Reply:
Raw input isn't restricted by window and it doesn't relate on screen pixels/window size.
Moreover, it's not always mouse input, it can be any device. I tried to notice that facts in docs.